update_command.php

```diff
@@ -94,34 +94,41 @@
         } else {
             // get all branches
             $allBranches = explode("\n", cmd('git branch -r', $MODULE_DIR));
+            $allBranches = array_filter($allBranches, fn($branch) => !str_contains($branch, 'HEAD ->'));
             $allBranches = array_map(fn($branch) => trim(str_replace('origin/', '', $branch)), $allBranches);
+            // reset index
+            $allBranches = array_values($allBranches);
```

Comment:
Why? What does this actually do? What does "reset index" mean?

Reply:
It's because the array_filter() call above will remove some values, though it doesn't reset the int based index

array_values([1 => 'a', 4 => 'b']) becomes [0 = > 'a', 1 => 'b'];

Reply:
Ahh, so "index" here is referring to the index of the array. Fair enough.